akka-docs/src/main/paradox/typed/actor-discovery.md

```diff
@@ -14,29 +14,37 @@ To use Akka Actor Typed, you must add the following dependency in your project:
 
 ## Obtaining Actor references
 
-There are two general ways to obtain @ref:[Actor references](../general/addressing.md#what-is-an-actor-reference): by
-@ref:[creating actors](actor-lifecycle.md#creating-actors) and by discovery using the @ref:[Receptionist](#receptionist).
-
-You can pass actor references between actors as constructor parameters or part of messages.
-
-Sometimes you need something to bootstrap the interaction, for example when actors are running on
-different nodes in the Cluster or when "dependency injection" with constructor parameters is not
-applicable.
+In order for an actor to send a message to one or more other actors it needs to obtain an @ref:[Actor reference](../general/addressing.md#what-is-an-actor-reference). 
+This reference relates to the notion of locality.
+In his 2010 paper, [Actor Model of Computation: Scalable Robust Information Systems](https://arxiv.org/abs/1008.1459), 
+Carl Hewitt relays locality and message processing as, 
+“an Actor can send messages only to addresses for which it has information by the following means: 
+1. that it receives in the message 
+2. that it already had before it received the message 
+3. that it creates while processing the message. “
```

Comment:
I'm not convinced we should go into theory here, I like the old hands on style better. Is there no there place in the docs where we do some more theory/background section where we already mention this that we could potentially crosslink instead?